README.md

```diff
@@ -552,6 +553,22 @@ This will produce elasticsearch output that looks like this:
 
 Note that the flattener does not deal with arrays at this time.
 
+### Generate Hash ID
+
+Fluentd re-emits events that failed to be indexed/ingested in Elasticsearch with a new and unique `_id` value, this means that congested Elasticsearch clusters that reject events (due to command queue overflow, for example) will cause Fluentd to re-emit the event with a new `_id`.
+
+Using generated hash id is another fix way to prevent duplicate `_id` in elasticsearch records.
+Here is a sample config:
+
+```
+<hash>
+  hash_type sha256   # md5, sha1, sha256, sha512 are available here.
+  hash_id_key _id    # storing genrated hash id key
+<hash>
+```
+
+:warning: In order to avoid hash-collisions and loosing data careful consideration is required when choosing the keys in the event record that should be used to calculate the hash.
```

Comment:
Seems like this warning can be removed now.

Reply:
👍